src/components/elements/cycleHyperlink.tsx

```diff
@@ -0,0 +1,130 @@
+import type {Stories} from '../../components/types/stories';
+
+export const CycleHyperlink = (props: any) => {
+  const story = props.story;
+  const all_stories = props.all_stories;
+
+  let prev_text = "<< Read previous part of story";
```

Comment:
I think these should be of type const and moved to the top of the file. Naming convention for these types of variables are usually all capital separated by underscores